### ISSUE TITLE
Metadata not fully documented

### ISSUE THREAD

**Issue by ChrisChinchilla** [EventStore/docs.geteventstore.com#265]
For more details see discussion here - https://github.com/EventStore/docs.geteventstore.com/pull/126

**Comment by alexeyzimarev** [comment]
Yeah, it would be nice to get `$correlationId` and `$causationId` reserved metadata fields mentioned there as well
